docs/hydra/guides/jwt.mdx

```diff
@@ -1,61 +1,60 @@
+ELECTRO can then use the access token to cash the monthly payments from the online bank.
 
 ## Ory JWT profile capabilities
 
-Ory supports both methods described in [RFC 7523](https://www.rfc-editor.org/rfc/rfc7523) for using JWTs as authorization grants
-and client authentication:
+Ory supports both methods for using JWTs as authorization grants and client authentication described in
+[RFC 7523](https://www.rfc-editor.org/rfc/rfc7523):
 
-- Using JWTs as Authorization Grants: Allows exchanging a JSON Web Token for an access token using an established trust
-  relationship.
-- Using JWTs for Client Authentication: Allows OAuth 2.0 client authentication using public/private keys via JSON Web Tokens.
+- Using JWTs as authorization grants allows exchanging JSON Web Tokens for access tokens using an established trust relationship.
+- Using JWTs for client authenticationa allows OAuth 2.0 client authentication using public/private keys via JSON Web Tokens.
 
 ## Using JWTs as Authorization Grants
 
-To use the `urn:ietf:params:oauth:grant-type:jwt-bearer` Authorization Grant, the client must make an OAuth 2.0 Access Token
-Request using the following specific parameter values and encodings:
+To use the `urn:ietf:params:oauth:grant-type:jwt-bearer` Authorization Grant, the client must make an OAuth 2.0 access token
+request using the following specific parameter values and encodings:
 
 - `grant_type` value must be `urn:ietf:params:oauth:grant-type:jwt-bearer`.
 - The `assertion` parameter must contain a single JWT.
 
-The `scope` parameter may also be included to indicate the requested scope:
+You can also add the `scope` parameter to indicate the requested scope:
 
-```
+```curl
 POST /oauth2/token HTTP/1.1
 Host: {project.slug}.projects.oryapis.com
 Content-Type: application/x-www-form-urlencoded
 
 scope=foo+bar
 &grant_type=urn%3Aietf%3Aparams%3Aoauth%3Agrant-type%3Ajwt-bearer
 &assertion=eyJhbGciOiJFUzI1NiIsImtpZCI6IjE2In0.
-eyJpc3Mi[...omitted for brevity...].
-J9l-ZhwP[...omitted for brevity...]
+eyJpc3Mi[...].
+J9l-ZhwP[...]
 ```
 
 ### Establishing a trust relationship
 
-Before using this grant type, a trust relationship must be established in Ory OAuth2 and OpenID Connect. Trust relationships come
-in two flavors:
+Before using this grant type, a trust relationship must be established in Ory OAuth2 and OpenID Connect. There are two types of
+trust relationships:
 
 - Trust relationships restricted to a single subject. This means that the issuer is only allowed to sign JWTs for the trusted
   subject.
-- Trust relationships that allow issuing tokens for any subject. This may be useful for some cases (like implementing a Secure
-  Token Service), but gives the issuer the ability to impersonate any user, so this should only be done if the issuer is as
-  trusted as your own Ory OAuth2 and OpenID Connect instance.
+- Trust relationships that allow issuing tokens for any subject. While this can be useful to, for example, implement a secure
```

Comment:
```suggestion
- Trust relationships that allow issuing tokens for any subject. While this can be useful to implement a secure
```

The "for example" splits the infinitive here. Maybe just leave it out?

Reply:
Agreed. Thanks :)